### PR TITLE
George Ho - Independent Challenge: 1

### PR DESCRIPTION
**Pull request by georgehwho**
George Ho

### CODE REVIEW

File: lib/restaurant.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+class Restaurant
+  attr_reader :opening_time, :name, :dishes
+
+  def initialize(time, name)
+    @opening_time = time
+    @name = name
+    @dishes = []
+  end
+
+  def convert_time_int(time = opening_time)
```
Comment (memcmahon) [inline lib/restaurant.rb:10]: nice use of a default value here.

File: lib/restaurant.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def open_for_lunch?
+    return true if convert_time_int < 12
+    false
+  end
```
Comment (memcmahon) [inline lib/restaurant.rb:28]: This works, but is more typing than is strictly necessary.  You can rely on the return value of the comparison:

```ruby
def open_for_lunch?
  convert_time_int < 12
end
```

File: test/restaurant_test.rb [hunk 3]
```diff
@@ -11,38 +11,40 @@ def test_it_exists
...
   def test_it_has_closing_time
-    skip
+    # skip
     restaurant1 = Restaurant.new('10:00', 'Fuel Cafe')
     restaurant2 = Restaurant.new('16:00', 'Il Poggio')
 
     assert_equal '18:00', restaurant1.closing_time(8)
     assert_equal '23:00', restaurant2.closing_time(7)
+    assert_equal ':00', restaurant2.closing_time(8)
+    assert_equal '1:00', restaurant2.closing_time(9)
```
Comment (memcmahon) [inline test/restaurant_test.rb:43]: Good edge case additions

**Review by memcmahon — COMMENTED** [review]
Great Job! Take a look at my notes, and let me know if you have any questions.

Score: 4
